config/src/config/consensus_config.rs

```diff
@@ -62,12 +62,12 @@ impl Default for ConsensusConfig {
             // defaulting to under 0.5s to broadcast the proposal to 100 validators
             // over 1gbps link
             max_sending_block_bytes: 600 * 1024, // 600 KB
-            max_sending_block_bytes_quorum_store_override: 5 * 1024 * 1024, // 5MB
-            max_receiving_block_txns: 10000,
+            max_sending_block_bytes_quorum_store_override: 10 * 1024 * 1024, // 5MB
+            max_receiving_block_txns: 16000,
```

Comment:
Nit, maybe `MAX_SENDING_BLOCK_TXNS_QUORUM_STORE_OVERRIDE + 6000` in case we end up changing the sending block size

Reply:
Made it 1.5 times of the sending size.